.travis.yml

```diff
@@ -66,6 +66,16 @@ matrix:
           - binutils-dev
           - g++-5
           - clang-format-3.7
+    - env: BUILD_TYPE="Debug" WITH_BFD="yes" WITH_GCC_6="yes" WITH_COVERAGE="yes"
```

Comment:
Can you remove `WITH_COVERAGE=yes` here, as running coverage for gcc-6 is not different from gcc-5